packages/mesos/extra/dcos-mesos-slave-public.service

```diff
@@ -12,6 +12,7 @@ EnvironmentFile=/opt/mesosphere/environment
 EnvironmentFile=/opt/mesosphere/etc/mesos-slave-common
 EnvironmentFile=-/opt/mesosphere/etc/mesos-slave-common-extras
 EnvironmentFile=/opt/mesosphere/etc/mesos-slave-public
+EnvironmentFile=-/opt/mesosphere/etc/mesos-slave-public-extras
```

Comment:
But do you want to switch `/opt/mesosphere/etc/mesos-slave-public` and `/opt/mesosphere/etc/mesos-slave-common-extras` so that "downstream gets the last say"? Ideally nobody should put the same thing in both of them.